storage/storage.go

```diff
@@ -27,7 +27,7 @@ type SampleAppender interface {
 	// sending samples. Local storage implementations will only drop metrics
 	// upon unrecoverable errors. Reporting any errors is done via metrics
 	// and logs and not the concern of the caller.
-	Append(*model.Sample)
+	Append(*model.Sample) error
```

Comment:
Doc string has to be adjusted (logging errors _is_ now a concern of the caller).